scripts/parse_existing_cohort.py

```diff
@@ -205,6 +219,12 @@ def get_existing_external_sequence_ids(self, participant_map: dict[str, dict]):
 @click.option(
     '--include-participant-column', 'include_participant_column', is_flag=True
 )
+@click.option(
+    '--allow-missing-files',
+    'allow_missing_files',
+    is_flag=True,
+    help='Set this flag to parse manifests with missing data',
```

Comment:
Nit, can we be more specific with this comment? i.e. 
```suggestion
    help='Set this flag to parse/ingest sequencing groups with missing reads',
```